src/java/org/apache/cassandra/metrics/TableMetrics.java

```diff
@@ -280,7 +280,7 @@ private static Pair<Long, Long> totalNonSystemTablesSize(Predicate<SSTableReader
     {
         long total = 0;
         long filtered = 0;
-        for (String keyspace : Schema.instance.getNonSystemKeyspaces().names())
+        for (String keyspace : Schema.instance.distributedKeyspaces().names())
```

Comment:
It seems that we are lying for TableMetrics too :-(

Reply:
can we address that in 5.0 and change metrics a bit?

Reply:
It was just a statement. We can let that for later.